html/changelogs/superhats-PR-329.yml

```
@@ -34,4 +34,4 @@ delete-after: True
 # Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
 # Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
 changes: 
-  - tweak: "The Engi Polytool augment in the loadout screen is now restricted to engineering, research and prospector roles. You can still get an implanted tool from a roboticist as any role after the round starts."
+  - imageadd: "Added a bunch of new skirt options, courtesy of Valentine1488"
```

```suggestion
  - imageadd: "Added a bunch of new skirt options, courtesy of Valentine1488"
```
I would ask you make your own changelog than edit Vhbraz's. Their thing was reverted, yes. Record keeping doesn't change though.

---

Oh, that's not supposed to be here.

---

I don't know how that happened